File: 30-pretraga/najveci.js
const nadjiNajveci = niz => {
  let najveci = niz[0]
  for(let i = 1; i < niz.length; i++) {
    if(najveci < niz[i]) {
      najveci = niz[i]
    }
  }
  return najveci
}

const brojevi = [1, 5, 2, 5, 33, 44, 77, 6, 7, 8, 9, 0, 2, 55, 66]
console.log(nadjiNajveci(brojevi))

File: 30-pretraga/najveci.py
def najveci(niz):
    max = niz[0]
    for x in range(len(niz)-1):
        if niz[x] > max:
            max = niz[x]
    return max


print(najveci([54,26,93,17,77,31,44,55,20]))
